concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,6 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Sidwell-2021-116	Sidwell, Paul and Alves, Mark	2021		116	basic	English	Vietic		SidwellAlves2021		This concept list is taken from a dataset on Nicobarese languages. The authors have also indicated whether the concepts recur in [Swadesh's list of 100](:ref:Swadesh-1955-100) and [Leipzig-Jakarta list](:ref:Tadmor-2009-100).		
```

Comment:
In the NOTE column, you should probably `Nicobarese` to `Vietic`

Reply:
Please write `and the [Leipzig-Jakarta list]` (article needed here).

Reply:
What do you mean by "(article needed here)"? A citation for the corresponding list?